com.unity.formats.alembic/Editor/Importer/AlembicImporter.cs

```diff
@@ -203,6 +211,27 @@ public override void OnImportAsset(AssetImportContext ctx)
             firstImport = false;
         }
 
+        [InitializeOnLoadMethod]
+        static void InitializeEditorCallback()
+        {
+            EditorApplication.update += DirtyCustomDependencies; //
```

Comment:
You should call DirtyCustomDependencies once here so it has a chance to setup the dependency value before the first import of the project assets starts.
This would prevent the double import of the alembic assets when opening a project after updating Unity or with the Library folder removed.